packages/cashier/src/components/cashier-locked/cashier-locked.tsx

```diff
@@ -1,35 +1,47 @@
 import React from 'react';
+import { useHistory } from 'react-router';
 import { useStore, observer } from '@deriv/stores';
 import { useCashierLocked, useDepositLocked, useIsSystemMaintenance } from '@deriv/hooks';
 import EmptyState from 'Components/empty-state';
 import getMessage from './cashier-locked-provider';
+import './cashier-locked.scss';
 
 const CashierLocked = observer(() => {
     const { client } = useStore();
     const {
         account_status,
         accounts,
         current_currency_type,
+        is_eu,
         is_withdrawal_lock: is_withdrawal_locked,
         loginid,
         is_identity_verification_needed,
+        mf_account_status,
     } = client;
     const is_cashier_locked = useCashierLocked();
     const is_system_maintenance = useIsSystemMaintenance();
     const is_deposit_locked = useDepositLocked();
+    const history = useHistory();
 
     const state = getMessage({
         cashier_validation: account_status.cashier_validation,
+        excluded_until: loginid ? accounts[loginid]?.excluded_until : undefined,
+        history,
         is_crypto: current_currency_type === 'crypto',
+        is_eu,
         is_system_maintenance,
         is_cashier_locked,
         is_deposit_locked,
         is_withdrawal_locked,
         is_identity_verification_needed,
-        excluded_until: loginid ? accounts[loginid]?.excluded_until : undefined,
+        is_pending_verification: mf_account_status === 'pending',
     });
 
-    return <EmptyState icon={state.icon} title={state.title} description={state.description} />;
+    return (
+        <div className='cashier-locked'>
```

Comment:
Hi @farhan-nurzi-deriv, can we use `<PageContainer />` over here?
```suggestion
        <PageContainer hide_breadcrumb>
            <EmptyState icon={state.icon} title={state.title} description={state.description} action={state.action} />
        </PageContainer>
```

Reply:
This will also eliminate the need for the style file.

Reply:
@aum-deriv i dont think we need to add `PageContainer`, because currently there is no breadcumb there, i added the style to remove the margin for the action button so your suggestion is for different case  🙏

Reply:
Initially i added style file to remove extra margin between the message and button, but when i added a new div suddenly i got extra margin on top. Thats why i also added `margin: 0`

<img width="770" alt="Screenshot 2023-08-03 at 3 34 00 PM" src="https://github.com/suisin-deriv/deriv-app/assets/125247833/899b6b70-a196-4142-af11-01b0dbe7dd38">

Reply:
> @aum-deriv i dont think we need to add `PageContainer`, because currently there is no breadcumb there, i added the style to remove the margin for the action button so your suggestion is for different case 🙏

@farhan-nurzi-deriv, I got your second point for the margin 👍 .